website/docs/ide-config.md

```diff
@@ -0,0 +1,62 @@
+<h2 id="vscode">VS Code</h2>
+
+> VS Code Extensions
+
+- ESLint
+  - id: "dbaeumer.vscode-eslint"
+  - recommend reason: "Integrates ESLint JavaScript into VS Code."
+- Stylelint
+  - id: "stylelint.vscode-stylelint"
+  - recommend reason: "Official Stylelint extension for Visual Studio Code"
+- GitLens
+  - id: "eamodio.gitlens"
+  - recommend reason: "It helps you to visualize code authorship at a glance via Git blame annotations and CodeLens"
+- markdownlint
+  - id: "DavidAnson.vscode-markdownlint"
+  - recommend reason: "Markdown linting and style checking for Visual Studio Code"
+
+> VS Code Settings
+
+```json5
+{
+  // ===
+  // Spacing
+  // ===
+
+  "editor.insertSpaces": true,
+  "editor.tabSize": 2,
+  "editor.trimAutoWhitespace": true,
+  "files.trimTrailingWhitespace": true,
+  "files.eol": "\n",
+  "files.insertFinalNewline": true,
+  "files.trimFinalNewlines": true,
+
+  // ===
+  // Files
+  // ===
+
+  "files.exclude": {
+    "**/dist": true
```

Comment:
Does this rule cover all the files that need to be excluded?

Reply:
Configure glob patterns for excluding files and folders in fulltext searches and quick open. Inherits all glob patterns from the `files.exclude` setting.

https://code.visualstudio.com/docs/getstarted/settings

Reply:
@barchielAbyss I mean, we already have .gitignore, do we really need this configuration? And we don't seem to be generating any folders named dist?